### PR TITLE
Added python requirements, virtual enviroment and script

### PR DESCRIPTION
**Pull request by Jaxobc**

### CODE REVIEW

**Review by kennzoid — CHANGES_REQUESTED** [review]
Going to jump in here and add some quick feedback since it's after work hours for the others.

I believe you are missing the requirements.txt file that's mentioned in the PR title. Also, these python files shouldn't be nested under src/main/**java**. Instead, we can just create a folder under src/main called "data_processing" or something descriptive like that.

**Review by briafassler9 — DISMISSED** [review]

File: portfolio/src/main/data_proccessing/bin/pyrsa-decrypt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python
```
Comment (kennzoid) [inline portfolio/src/main/data_proccessing/bin/pyrsa-decrypt]: What are these pyrsa-*.py files for?

File: portfolio/src/main/java/com/google/sps/scripts/processor.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print("python_script works")
```
Comment (kennzoid) [inline portfolio/src/main/java/com/google/sps/scripts/processor.py]: Is this still needed?

Comment (Jaxobc) [reply portfolio/src/main/java/com/google/sps/scripts/processor.py]: No it is no longer necessary and I will remove it.

File: portfolio/src/main/data_proccessing/bin/pyrsa-decrypt (hunk 1 above)
Comment (Jaxobc) [reply portfolio/src/main/data_proccessing/bin/pyrsa-decrypt]: I really do not know. I tried multiple things to update my python so I think they may have appeared as a result of that. I'll remove them and see what happens.

**Review by kennzoid — APPROVED** [review]

**Review by briafassler9 — APPROVED** [review]

**Pull request merged**